lib/controllers/github.js

```diff
@@ -7,14 +7,25 @@ const {
   getGithubProfile,
 } = require('../services/githubServices');
 const authenticate = require('../middleware/authenticate');
-
 const ONE_DAY_IN_MS = 1000 * 60 * 60 * 24;
-
+const IS_DEPLOYED = process.env.NODE_ENV === 'production';
 module.exports = Router()
+  .delete('/sessions', (req, res) => {
+    console.log('req.body', req.body);
+    res
+      .clearCookie(process.env.COOKIE_NAME, {
+        httpOnly: true,
+        secure: IS_DEPLOYED,
+        sameSite: IS_DEPLOYED ? 'none' : 'strict',
+        maxAge: ONE_DAY_IN_MS,
+      })
+      .status(204)
+      .send();
+  })
   .get('/login', async (req, res) => {
     try {
       res.redirect(
-        `https://github.com/login/oauth/authorize?client_id=${process.env.GITHUB_CLIENT_ID}&scope=user&redirect_uri=${process.env.GITHUB_REDIRECT_URI}`
+        `https://github.com/login/oauth/authorize?client_id=${process.env.CLIENT_ID}&scope=user&redirect_uri=${process.env.REDIRECT_URI}`
       );
     } catch (error) {
       console.log(error);
```

Comment:
It appears our site redirects to localhost instead of deployed site. We will want to create another github oauth app that uses our deployed site instead of localhost.